sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Integration.Tests.AspNetCoreWebApp/Azure.Monitor.OpenTelemetry.Exporter.Integration.Tests.AspNetCoreWebApp.csproj

```diff
@@ -11,6 +11,8 @@
   </PropertyGroup>
 
   <ItemGroup>
```

Comment:
It may be worth a comment here and search mentioning why the override is being applied.

Reply:
Since this is a T2 library and a test project, should we be concerned with hoisting up to 4.7.2?

Reply:
It was using 4.5.0 before and should be upgraded to 4.5.1 per guidance. Pulling in 4.7.2 would pull in far more than what it was built for originally. The owner can do that if they see a benefit.